lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1713,7 +1713,10 @@ ParseResult ConstantIntOp::parse(OpAsmParser &parser, OperationState &result) {
 
 void ConstantIntOp::print(OpAsmPrinter &p) {
   p << " ";
-  p << getValue().getSExtValue();
+  if (getValueAttr().getType().isUnsignedInteger())
```

Comment:
The intention is that this attribute is always `i64`. Can you update the verifier to reject any attribute that is not `i64` in type? We shouldn't be hitting this case at all -- how are you running into it?

Reply:
Sure, I can update the verifier. 
I actually didn't run into `uint` case. Mine case was a folding a `torch_c.toi64` with `si64` attribute `torch.constant.int`. My intention is to remove the signedness when folding. I think torch integer types by default are signed integers according to https://github.com/llvm/torch-mlir/blob/main/include/torch-mlir/Dialect/Torch/IR/TorchTypes.td#L127

Reply:
I think `torch.constant.int` should always have a signless integer attribute -- https://github.com/llvm/torch-mlir/blob/eaaaeb6ff15d676f901035f86af55a17dc252781/python/torch_mlir/dialects/torch/importer/jit_ir/csrc/torch_to_mlir_utils.cpp#L391

For now we want the verifier to enforce that I think. Where are you seeing si64? The case for tensor dtype is a bit different so TorchTypes.td#L127 doesn't apply here. Here we are talking about the modeling of Python scalars.